code/modules/clothing/clothing.dm

```diff
@@ -599,6 +599,7 @@
 /obj/item/clothing/mask/proc/adjust_mask(mob/user, var/self = TRUE)
 	set name = "Adjust Mask"
 	set category = "Object"
+	set src in usr
 
 	if(!adjustable)
```

Comment:
can you make it so that masks that aren't adjustable don't get the action button thnx bb